docs/use-github-in-bcgov/transferring-repos-to-bc-gov-orgs.md

```diff
@@ -0,0 +1,102 @@
+# Transferring repositories
+
+This page describes how to transfer repositories from a team or ministry's GitHub organization to either the [bcgov](https://github.com/bcgov) or [bcgov-c](https://github.com/bcgov-c) GitHub organization.
+
+Learn more about the [bcgov](https://github.com/bcgov) and [bcgov-c](https://github.com/bcgov-c) organizations in the [B.C. Government organizations in GitHub](bc-government-organizations-in-github.md) guide.
+
+# Will it be hard?
+
+Short answer – no! 🎉
+
+For transfers from another GitHub organization, GitHub fortunately makes repository transfers very easy. There are a few details you’ll need to take care of for most repositories (e.g., authorizing PATs, updating team access) and some other considerations that won’t apply to everyone (e.g., packages).
+
+The bulk of the work will revolve around updating and validating any deployment pipelines that reference your repositories.
+
+Regardless of your repository source, take care to isolate changes to your development/test deployment environment and validate before promoting to production.
+
+# Recommendations (read before you begin)
+
+The GitHub transfer function will:
+
+* Transfer issues, pull requests, wiki, stars, and watchers
+* Maintain associations for webhooks, services, secrets, forks, and deploy keys
+* Automatically redirect links and Git commands from the old repository to the new one
+* **Exception:** Packages aren't transferred – see [Package management](#package-management) below
+
+Review GitHub's [Transferring a repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository) documentation for a complete list of things that are transferred. Make note of the items that are not transferred.
+
+## Setup
+
+* Ensure all team members have joined the destination organization:
+  * [How to join the bcgov organization](bc-government-organizations-in-github.md/#directions-to-sign-up-and-link-your-account-for-bcgov)
+  * [How to join the bcgov-c organization](bc-government-organizations-in-github.md/#directions-to-sign-up-and-link-your-account-for-bcgov-c)
+* Note: Outside collaborators aren't allowed in the [bcgov](https://github.com/bcgov) and [bcgov-c](https://github.com/bcgov-c) organizations.
+* Users with individual access will retain the same level of access, provided they are members of the destination organization.
+  * For transfers to [bcgov](https://github.com/bcgov), ensure at least one individual with 'admin' access on your repository is a member of the [bcgov](https://github.com/bcgov) organization. This will prevent loss of admin access after the transfer.
```

Comment:
For transfers to [bcgov](https://github.com/bcgov), make sure at least one individual with 'admin' access on your repository is a member of the [bcgov](https://github.com/bcgov) organization. The access prevents loss of admin access after the transfer.